### ISSUE TITLE
Not able to get results on local development.

### ISSUE THREAD

**Issue by securit** [osiset/Basic-Shopify-API#73]
I'm not seeing responses from Guzzle on my local dev machine, but I am from production.

In the simplest of terms, I have the same code which I can use in Tinker.
```
use Osiset\BasicShopifyAPI\BasicShopifyAPI;
use Osiset\BasicShopifyAPI\Options;
use Osiset\BasicShopifyAPI\Session;

$options = new Options();
$options->setVersion(env('SHOPIFY_API_VERSION'));
$options->setGuzzleOptions(['connect_timeout' => 10, 'debug' => true]);
$api = new BasicShopifyAPI($options);
$api->setSession(new Session(env('SHOPIFY_SHOP_DOMAIN'), env('SHOPIFY_API_PASSWORD')));
$result = $api->graph('{ shop { name } }');
```

On production I get a full Guzzle debug as expected

On development I get 

```
* Resolving timed out after 10006 milliseconds
* Closing connection 0
=> [
     "errors" => true,
     "response" => null,
     "status" => null,
     "body" => null,
     "exception" => GuzzleHttp\Exception\ConnectException {#3937
       #message: "cURL error 28: Resolving timed out after 10006 milliseconds (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)",
       #file: "/Users/greg/PhpStorm/enversion/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php",
       #line: 200,
     },
     "timestamps" => [
       1594191186.6871,
     ],
   ]
```

Yet if I run the same GraphQL using curl on the development environment I get what I expect.

```
curl -X POST "https://######.myshopify.com/admin/api/2020-07/graphql.json" -H "Content-Type: application/graphql" -H "X-Shopify-Access-Token: ########" -d '{ shop{ name } }'
{"data":{"shop":{"name":"########"}},"extensions":{"cost":{"requestedQueryCost":1,"actualQueryCost":1,"throttleStatus":{"maximumAvailable":1000.0,"currentlyAvailable":999,"restoreRate":50.0}}}}%
```

On the development the only difference in the logs are session specific values or the `TCP_NODELAY set` is this timing out because of TCP's Nagle algorithm on this connection? 

This dev environment is Laravel Valet.

**Comment by osiset** [comment]
This is most likely outside my realm to fix. This is probably an issue with php-curl.

Your request is timing out after 10 seconds.

I will do some digging, but I don't think there will be anything I can do on my side.. I simply utilize Guzzle which by default is using cURL backend.

What happens if you run a php curl request?

**Comment by securit** [comment]
You are correct, for some reason from Valet it takes 30 seconds before it gets a response. Can I increase the timeout here?

**Comment by securit** [comment]
I have gotten to the bottom of this issue.

Firstly there were some expired digital certificates in my keychain. I removed all of the expired certificates one by boring one - I say boring because it required me to re enter my system password for each and every one... my system password is very long

I ran a openssl s_client -connect www.google.com:443 and got different results for google vs shopify.

I checked the DNS settings and discovered I did not have a setting for 127.0.0.1. So I put that in after my two other DNS server settings (one points to my local router, the other to Google). There was no improvement.

However, once I moved the 127.0.0.1 to the top of the list "Hey Presto!" :)

I would say that the google response was different because it was resolving google via the 8.8.8.8 DNS entry.

It sounds like dnsmasq requires a 127.0.0.1 entry at the top of the list of DNS resolvers in order to function correctly.
